polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala

```diff
@@ -103,7 +105,19 @@ trait FileBasedRepository extends NotebookRepository[IO] {
               }
             },
             content => {
-              writeString(extPath, content)
+              if (relativePath.endsWith(".json")) { // assume zeppelin
```

Comment:
😬

Reply:
yeah... any better ideas? I figured this was better than a whole other code path just for zeppelin... :-\

Reply:
Zeppelin is really all around not a good citizen (as much as I prefer it to Jupyter). At least make it `.zeppelin.json` or something, ya jerks

Reply:
Sorry not code path but user-interaction-path: somehow we need to tell whether this is a zeppelin file. We could do it in the UI by having a separate button or something but I figured it was better to kind of guess here. Definitely open to other ideas if you have any!